### PR TITLE
fix: race condition in tests

### PR DESCRIPTION
**Pull request by krzysztofdrys**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,9 +21,13 @@ up: ## run all applications in stack
 	docker compose up -d
 
 .PHONY: unit-tests
-unit-tests: ## run unit tests without e2e-tests directory..
+unit-tests: ## run unit tests without e2e-tests directory.
 	go test -race -count=1 `go list ./... | grep -v e2e-tests`
 
+.PHONY: unit-tests-ci
+unit-tests-ci: ## run unit tests without e2e-tests directory (multiple times to find race conditions).
+	go test -race -count=50 `go list ./... | grep -v e2e-tests`
```
Comment (tobiaszheller) [inline Makefile:29]: You can pass failfast flag also

File: internal/websocket/common/hub_pool_test.go [hunk 2]
```diff
@@ -54,10 +54,11 @@ func TestCreateRemoveConcurrently(t *testing.T) {
 	hubs := &sync.Map{}
 
 	wg := sync.WaitGroup{}
-	wg.Add(channelsNo * clientsPerChannel)
+	wg.Add(channelsNo*clientsPerChannel + channelsNo)
```
Comment (tobiaszheller) [inline internal/websocket/common/hub_pool_test.go:57]: Caan you explain why this logic? Preferably in comment in code.

Comment (krzysztofdrys) [reply internal/websocket/common/hub_pool_test.go:60]: Explained in comment

File: Makefile (hunk 1 above)
Comment (krzysztofdrys) [reply Makefile]: Done

**Review by tobiaszheller — APPROVED** [review]

**Comment by hamiamiri6** [comment]
> 

https://github.com/twofas/2fas-server/commit/7f7ea0693a0b53966e75916157b28d4094f32250

**Pull request merged**